.travis.yml

```diff
@@ -26,8 +28,19 @@ jobs:
           dist: xenial
           compiler: clang
           name: "Ubuntu 16.04 LTS - Clang"
+        - os: linux
+          dist: bionic
+          compiler: gcc
+          name: "Ubuntu 18.04 LTS - GCC"
+        - os: linux
+          dist: bionic
+          compiler: clang
+          name: "Ubuntu 18.04 LTS - Clang"
```

Comment:
Those are unnecessary as we included `dist: bionic`. so it is built automatically for both compilers.

Reply:
perhaps...but it'd probably be good to keep so that we can make it easy to roll forward with new editions, such as when Travis-CI supports 20.04.

Reply:
The issue is that we're running those tests twice 😅 
I'd suggest to add a comment next to `dist: bionic` that once updated, the old dist needs to be added manually to the `jobs` section.
I'll see if I can file a feature request with Travis-CI to add distros to the matrix like you add OSes.